examples/c5g7_transient/rod_translation/pin_cells_mg.py

```diff
@@ -0,0 +1,115 @@
+import openmc
```

Comment:
File is identical to `density_change/pin_cells_mg.py`. Consider removing and adding a symbolic link instead